Base/usr/share/man/man1/stat.md

```diff
@@ -0,0 +1,36 @@
+## Name
+
+stat
+
+## Synopsis
+
+```**sh**
+$ stat [-L] file
+```
+
+## Description
+
+Display file status.
+
+## Options:
+
+* `-L`: Follow links to files
+* `--help`: Display help message and exit
+* `--version`: Print version
+
+## Examples
+
+```sh
+$ stat -L /bin/ls
+File: /bin/ls
+Inode: 8288
+Size: 184896
+Links: 1
+Blocks: 376
+UID: 0 (root)
+GID: 0 (root)
+Mode: (100755)/-rwxr-xr-x)
+Accessed: 2022-10-20 03:56:56
+Modified: 2022-10-20 03:55:34
+Changed: 2022-10-20 03:56:57
```

Comment:
I'd say lets go with what serenity actually outputs here

```suggestion
$ stat -L /bin/ls
    File: /bin/ls
   Inode: 8288
    Size: 184896
   Links: 1
  Blocks: 376
     UID: 0 (root)
     GID: 0 (root)
    Mode: (100755/-rwxr-xr-x)
Accessed: 2022-10-20 03:56:56
Modified: 2022-10-20 03:55:34
 Changed: 2022-10-20 03:56:57
```

Reply:
Alright, I manually typed in what I saw in Serenity, but will keep aware of the spacing of the output.

Reply:
Alright, I manually typed in what I saw in Serenity, but will keep aware of the spacing of the output. What do I need to do to fix this so that this PR can be merged?

Reply:
If you edit the codeblock to be the exact output of the command (including leading spaces) I will approve the pr and link it on discord so a maintainer can take a look. Although you are encouraged to join the discord and advertise the pr yourself